signing/src/main/java/tech/pegasys/web3signer/signing/secp256k1/aws/AwsKMSSigner.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2023 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.signing.secp256k1.aws;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import tech.pegasys.web3signer.signing.config.AwsCredentialsProviderFactory;
+import tech.pegasys.web3signer.signing.config.metadata.AwsKMSMetadata;
+import tech.pegasys.web3signer.signing.secp256k1.Signature;
+import tech.pegasys.web3signer.signing.secp256k1.Signer;
+import tech.pegasys.web3signer.signing.secp256k1.util.Eth1SignatureUtil;
+
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.interfaces.ECPublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.X509EncodedKeySpec;
+
+import org.web3j.crypto.Hash;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.core.SdkBytes;
+import software.amazon.awssdk.services.kms.KmsClient;
+import software.amazon.awssdk.services.kms.model.GetPublicKeyRequest;
+import software.amazon.awssdk.services.kms.model.GetPublicKeyResponse;
+import software.amazon.awssdk.services.kms.model.KeySpec;
+import software.amazon.awssdk.services.kms.model.MessageType;
+import software.amazon.awssdk.services.kms.model.SignRequest;
+import software.amazon.awssdk.services.kms.model.SigningAlgorithmSpec;
+
+public class AwsKMSSigner implements Signer {
```

Comment:
nit: inconsistent initialisation capitalisation (AwsKmsSigner I think is convention over AWSKMSSigner)

Reply:
👍🏼. Refactored the code to use `AwsKms...` naming convention.